code/mathematical-algorithms/armstrong_numbers/armstrong_numbers.java

```diff
@@ -0,0 +1,21 @@
+import java.util.Scanner;
+
+class Main {
+	public static void main(String args[]) {
+		Scanner in = new Scanner(System.in);
+
+		System.out.println("Enter a 3-digit number : ");
+		int number = in.nextInt();
+		int a = number % 10;
+		int b = (number / 10) % 10;
+		int c = number / 100;
+
+		if ((a*a*a + b*b*b + c*c*c) == number) {
+			System.out.println(number + " is a armstring number.");
```

Comment:
You have a typo here. Should be `armstrong`, not `armstring`

Reply:
Updated.